### PR TITLE
Hide the attachments accordion on the lesson details page if no attachments are available

### LINKED ISSUE #2611: (SP: 1) Hide the attachments accordion on the lesson details page if no attachments are available
The attachments section should only be visible when there are attachments to display. If no attachments are present, the accordion should be hidden to improve the user experience and reduce unnecessary UI elements.

image

### PR DESCRIPTION
**Pull request by BohdanMylyi**
Expected Result:
The attachments section should only be visible when there are attachments to display. If no attachments are present, the accordion should be hidden to improve the user experience and reduce unnecessary UI elements.

Actual Result:

https://github.com/user-attachments/assets/55270cca-bbda-45e7-b454-235a4c875037

### CODE REVIEW

**Review by PavloDolia — APPROVED** [review]

File: src/pages/lesson-details/LessonDetails.tsx
Comment (dudchakk) [inline src/pages/lesson-details/LessonDetails.tsx:1]: why did you remove {} in row 55?

**Review by dudchakk — APPROVED** [review]

**Review by ipasic-softserve — APPROVED** [review]
